lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -30,12 +40,19 @@ export default function FilePickerApp({ onSubmit }) {
     filePicker: {
       formAction,
       formFields,
-      canvas: { ltiLaunchUrl },
+      canvas: { groupsEnabled: enableGroupConfig, ltiLaunchUrl },
```

Comment:
I aliased `groupsEnabled` to `enableGroupConfig` locally to clarify what this property means within the `FilePickerApp` component: It doesn't mean that groups/group sets are enabled for the current assignment, it just means that using group sets is supported for the LMS app installation.